### PR TITLE
va-modal: button widths, margin bug fix, storybook copy, title font family

### PR DESCRIPTION
**Pull request by powellkerry**
## Chromatic

https://2116-modal-storybook--60f9b557105290003b387cd5.chromatic.com

## Description
- Update to make the buttons be full width in mobile
- Make the storybook copy/content match sketch
- Update title font to be bitter
- Fix bug that was making the margins of the modal wider than they should be

Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2116

## Testing done
Local testing in Chrome, Safari, Edge, and Firefox

## Screenshots

Before:
Screenshot 2023-09-28 at 9 41 51 AM

After:

Screenshot 2023-09-28 at 9 42 26 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by jamigibbs — APPROVED** [review]
Dev looks good to me

File: packages/storybook/stories/va-modal-uswds.stories.jsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ const Template = ({
         visible={isVisible}
       >
         <p>
-          You have unsaved changes that will be lost.
+          This is a succinct, helful {status} message
```
Comment (micahchiang) [inline packages/storybook/stories/va-modal-uswds.stories.jsx:83]:
```suggestion
          This is a succinct, helpful {status} message
```

**Review by micahchiang — APPROVED** [review]
Looks good just recommended a typo change.

**Review by babsdenney — APPROVED** [review]
The fixes look great! Thank you!

**Pull request merged**